.circleci/src/workflows/web.yml

```diff
@@ -52,10 +52,21 @@ jobs:
         branches:
           only: /(^release.*)$/
 
+  - web-test-staging-essential:
```

Comment:
probably should rename both of these to `web-test-cypress-pr` and `web-test-cypress-main` or something

Reply:
yeah I was on the fence about the naming here, I think I might go with your suggestion and just name things PR or no

Reply:
I'm fine with the way you have it tbh, `@essential` is a good token to grep for and it makes sense to use it consistently all the way through the ci workflow name